packages/cli-kit/src/public/node/themes/api.ts

```diff
@@ -51,7 +51,7 @@ export async function deleteThemeAsset(id: number, key: Key, session: AdminSessi
   const response = await request('DELETE', `/themes/${id}/assets`, session, undefined, {
     'asset[key]': key,
   })
-  return Boolean(response.json.message)
```

Comment:
I've validated with the core API that this should be safe.

The `asset api` will return a `200` if the asset is deleted, but will also return a `200` if an asset already does not exist. The latter case would include an empty message.

I believe I initially made this decision to provide us with the ability to distinguish between the two cases. Seems like it may not be necessary. 

cc: @karreiro